README.md

```diff
@@ -85,6 +85,10 @@ See [§ Sponsor][sponsor] on our site for how to help financially.
 </tr>
 </table>
 
+## Contributing
+
+See the `CONTRIBUTING.md` in the `packages` subdirectories that have it.
+
```

Comment:
None of the packages have a contributing document.

```suggestion
```

Reply:
I added one for the VS Code extension that's why I added this. Should I remove it?

Reply:
Did you mean you added `.github/README.md`? Anyway, yes, please remove it. :)

Reply:
Yeah - but just removed like you said.